shared/common-adapters/icon.desktop.js

```diff
@@ -9,7 +10,16 @@ import {resolveImageAsURL} from '../../desktop/resolve-root'
 import type {Exact} from '../constants/types/more'
 import type {Props} from './icon'
 
-class Icon extends PureComponent<void, Exact<Props>, void> {
+class Icon extends Component<void, Exact<Props>, void> {
+  shouldComponentUpdate (nextProps: Exact<Props>, nextState: any): boolean {
```

Comment:
Do a shallow compare, but another shallow compare on the style attribute. 99% of the time the type / style changes but ultimately is the equivalent object